proto/evm/v1beta1/genesis.proto

```diff
@@ -4,10 +4,28 @@ option go_package = "github.com/axelarnetwork/axelar-core/x/evm/types";
 
 import "gogoproto/gogo.proto";
 import "evm/v1beta1/params.proto";
+import "evm/v1beta1/types.proto";
 
 option (gogoproto.goproto_getters_all) = false;
 
 // GenesisState represents the genesis state
 message GenesisState {
-  repeated Params params = 1 [ (gogoproto.nullable) = false ];
-}
\ No newline at end of file
+  option (gogoproto.stable_marshaler) = true;
```

Comment:
Does evm not have any module-wise param anymore? I'm thinking `voting_threshold` and `min_voter_count`. Why make them chain-specific?

Reply:
I didn't change any logic here, I just went from `[]Params{chainparams1, ...}` to `[]Chain{chain1{params}, ...}`